README.md

```diff
@@ -7,13 +7,25 @@
 
 # Taichi 官方中文文档
 
-预览地址: https://taichi.readthedocs.io/zh_CN/latest/
+[预览地址](https://taichi.readthedocs.io/zh_CN/latest/)
+
+[Taichi](https://github.com/taichi-dev/taichi) 是一门高效的、以 Python 语法封装的计算机图形应用语言。这里是 Taichi 的中文文档。
 
 |         | readthedocs.io | GitHub Actions |
-| ------- | -------------- | --------------- |
-| latest  | [![Documentation Status](https://readthedocs.org/projects/taichi-zh/badge/?version=latest)](https://taichi.readthedocs.io/zh_CN/latest/?badge=latest) | ![提交检查](https://img.shields.io/github/workflow/status/taichi-dev/taichi-docs-zh-cn/提交检查) |
+| ------- | -------------- | -------------- |
+| latest  | [![构建状态](https://readthedocs.org/projects/taichi-zh/badge/?version=latest)](https://taichi.readthedocs.io/zh_CN/latest/?badge=latest) | ![提交检查](https://img.shields.io/github/workflow/status/taichi-dev/taichi-docs-zh-cn/提交检查) |
 
-[Taichi](https://github.com/taichi-dev/taichi) 是一门高效的、以 Python 语法封装的计算机图形应用语言。这里是 Taichi 的中文文档。
+## 鸣谢
+
+Taichi 简体中文文档由 [Ark (StephenArk30)](https://github.com/StephenArk30) 创立，至今做出重大贡献者有：
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
+  - [Chengchen Wang (rexwangcc)](https://github.com/rexwangcc)
```

Comment:
Would you mind to display your Chinese name here? Please tell me if you want to do so :)

Reply:
Sure : -) Added mine below

Reply:
Thanks! @rexwangcc @StephenArk30 how about you?